tensorflow/core/ops/nn_ops.cc

```diff
@@ -214,7 +214,7 @@ REGISTER_OP("FusedBatchNormV3")
     .Output("reserve_space_2: U")
     .Output("reserve_space_3: U")
     .Attr("T: {half, bfloat16, float}")
-    .Attr("U: {float}")
+    .Attr("U: {half, float}")
```

Comment:
Why not support bfloat16 here, when it's supported in FusedBatchNormV2?

Reply:
i totally agree.  mistake.  will correct.